src/debug/migrate-ln-payments-trackpaymentsv2.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * how to run:
+ *	. ./.envrc && yarn ts-node \
+ *		--files \
+ *			-r tsconfig-paths/register \
+ *			-r src/services/tracing.ts \
+ *		src/debug/migrate-ln-payments-trackpaymentsv2.ts \
+ *      2>&1 | tee migration.logs
+ */
+
+import { isUp } from "@services/lnd/health"
+import { params as unauthParams } from "@services/lnd/unauth"
+import { PaymentNotFoundError, PaymentStatus } from "@domain/bitcoin/lightning"
+import { CouldNotFindError } from "@domain/errors"
+import { LedgerService } from "@services/ledger"
+import { LndService } from "@services/lnd"
+import { baseLogger } from "@services/logger"
+import { LnPaymentsRepository } from "@services/mongoose"
+import {
+  addAttributesToCurrentSpan,
+  addEventToCurrentSpan,
+  asyncRunInSpan,
+  SemanticAttributes,
+} from "@services/tracing"
+import { setupMongoConnection } from "@services/mongodb"
+
+let lndService
+
+export const main = async (): Promise<true> => {
+  lndService = LndService()
+  if (lndService instanceof Error) return true
+
+  const paymentHashes = await LedgerService().listAllPaymentHashes()
+  for await (const paymentHash of paymentHashes) {
+    const res = await migrateLnPayment(paymentHash)
+    if (res instanceof Error) break
+  }
+  return true
+}
+
+const migrateLnPayment = async (
+  paymentHash: PaymentHash | LightningServiceError,
+): Promise<true | LightningServiceError> =>
+  asyncRunInSpan(
+    "debug.migrateLnPayment",
+    {
+      [SemanticAttributes.CODE_FUNCTION]: "debug.migrateLnPayment",
```

Comment:
also, just for future spans... please add `debug` as namespace, `migrateLnPayment` as code function and `debug.migrateLnPayment` as a span name

Reply:
Did you mean like this? https://github.com/GaloyMoney/galoy/pull/1041/commits/cc9e9900ceafcdf08375d62af5698ee3aef8e245

Reply:
yes